backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -173,6 +182,125 @@ const testGetContainerStats = () => {
 	});
 };
 
+const formatRevisions = (revs, includeVoids = false) => {
+	let formattedRevisions = revs;
+
+	if (!includeVoids) {
+		formattedRevisions = formattedRevisions.filter((rev) => !rev.void);
+	}
+
+	formattedRevisions = formattedRevisions.map((rev) => ({ ...rev, timestamp: rev.timestamp.toISOString() }))
```

Comment:
if you use flatMap then you don't need to do a filter at L189

```suggestion
	formattedRevisions = formattedRevisions.flatMap((rev) => includeVoid || !rev.void ? { ...rev, timestamp: rev.timestamp.toISOString() } : []);
```